src/screen/TouchPadScreen.js

```diff
@@ -268,20 +272,49 @@ const TouchPadScreen = ({ navigation: { navigate }, route }) => {
   useFocusEffect(
     React.useCallback(() => {
       (async () => {
-        idToken.current = await AsyncStorage.getItem("idToken");
+        token.current = await AsyncStorage.getItem("idToken");
+        const idToken = await AsyncStorage.getItem("idToken");
         userCustom.current = await axios.get(
           `${SERVER_PORT}/users/${
-            JSON.parse(idToken.current).user.email
+            JSON.parse(token.current).user.email
           }/customGesture`,
+          {
+            headers: {
+              idToken: idToken,
+            },
+          },
         );
 
         gestureFunctions.current = await axios.get(
           `${SERVER_PORT}/users/${
-            JSON.parse(idToken.current).user.email
+            JSON.parse(token.current).user.email
           }/gestures`,
+          {
+            headers: {
+              idToken: idToken,
+            },
+          },
         );
       })();
       return () => {
+        (async () => {
+          token.current = await AsyncStorage.getItem("idToken");
+          const idToken = await AsyncStorage.getItem("idToken");
```

Comment:
```suggestion
          const idToken = await AsyncStorage.getItem("idToken");
          
          token.current = await AsyncStorage.getItem("idToken");
```
위처럼 변수의 선언이 더 위로 가면 보다 깔끔해보일것같습니다.